lib/dune

```diff
@@ -3,27 +3,21 @@
  (public_name io-page)
  (modules Io_page)
  (libraries cstruct bigarray-compat)
+ (c_flags :standard)
```

Comment:
what's the difference between ` (c_flags :standard)` and no `c_flags` entry at all?

Reply:
Actually there's no difference !

Reply:
so can this be removed?

Reply:
Indeed !